tests/fixtures/drafts/api-resource-pagination.yaml

```diff
@@ -0,0 +1,11 @@
+models:
+  Post:
+    title: string
+    body: text
+controllers:
+  Post:
+    resource: api
+    index:
+      query: all
```

Comment:
If we don't need to define `query:all` why is it here?

Reply:
It was in the original draft provided for the issue, I used it to make sure the issue was first addressed.

Reply:
Is there any value in leaving it? Seems from you PR description, it's no longer a valid combo.

Reply:
Yes. Its valid. As in, internally blueprint is “intelligently” defining the 'query: all', so when you ask for a paginated resource it will generate the correct `Model::paginate()`

Reply:
Sorry, I am a little confused. I guess I don't understand this item:

> No longer a need to define `query: all` when you define resource: 'paginate:{model}'.

Reply:
I misspoke. Does “shorthand” sound better?

Reply:
Yeah, makes more sense. I guess my ultimate point is, if there is a "shorthand" I just want to ensure there is some kind of test for it. So maybe this fixture could be it?

Reply:
Yes. This is an example of the one defining the `query: all`. We already had tests that do not include the query portion.